Makefile

```diff
@@ -403,3 +403,16 @@ consistent-crds-manifests-upstream:
 	$(Q)cd ./manifests-upstream/${OPERATOR_VERSION}/ && ln -srf ../../deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml \
 	servicebindingrequests.apps.openshift.io.crd.yaml
 
+## -- Target for merge to master dev release --
+
+.PHONY: build-image push-image
+## Make a dev release on every merge to master
+merge-to-master-release: 
+ 	$(eval COMMIT_COUNT := $(shell git rev-list --count HEAD))
+	$(Q)operator-sdk build \
```

Comment:
Which `Dockerfile` does this use?

Reply:
The one in root `/Dockerfile` is the one we are using in productization.

Reply:
for productization it is /Dockerfile.rhel . This one is under build/Dockerfile